utils/query_anasum_runparameters.py

```diff
@@ -0,0 +1,30 @@
+import uproot as up
+
+
+def get_epoch_effective_area(run):
+    '''
+    Reads and returns the epoch from the mscw log and the effective area file name
+    from the anasum log stored within a anasum.root file.
+
+    Parameters
+    ----------
+    run: int
+        VERITAS run number
+    Returns
+    -------
+    Tuple
+        epoch and effective area
+    '''
+    file = up.open(f"./{run}.anasum.root")
```

Comment:
Does this read a local file only? Can we change this so that the full file path (directory + file name) of the anasum file is the function argument?

Reply:
fixed